src/tasks/block.rs

```diff
@@ -347,7 +346,7 @@ impl Simulator {
             Some(basefee) => basefee,
             None => {
                 warn!("get basefee failed - RPC error likely occurred");
-                BASEFEE_DEFAULT
+                todo!()
```

Comment:
Is this intended in this commit?

Reply:
yes and explicitly flagged in the pr description. i would rather have a crash than faulty behavior